### DISCUSSION TITLE
10.0.22631.2129 not working?

### DISCUSSION THREAD

**Discussion by aldarund** [valinet/ExplorerPatcher#1767] [General]
Updated win to 10.0.22631.2129 and windows keep stacking, like patch not applied anymore

**Comment by pyrates999** [comment]
That's an insider build: https://www.elevenforum.com/t/kb5029359-windows-11-insider-beta-22621-2129-and-22631-2129-aug-2.16857/

When running insider builds, you must disable the symbols from being downloaded as insider builds usually include the debug symbols already: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

As for the windows keep stacking, what patch are you referring to?

**Reply by aldarund** [reply]
I mean in taskbar  all windows from one app e.g. chrome show up as a single entity as without patcher.
![image](https://github.com/valinet/ExplorerPatcher/assets/571159/0c399484-5238-4ca6-a103-7179e0683b08)
As i read that doesnt seem to be needing symbols

**Reply by pyrates999** [reply]
I would think Microsoft changed this behavior then.

Symbols are a technical requirement for explorer patcher to function.

**Reply by bgovern** [reply]
Same issue.  After that installed no functionality from EP.

**Comment by pyrates999** [comment]
> I mean in taskbar all windows from one app e.g. chrome show up as a single entity as without patcher. ![image](https://user-images.githubusercontent.com/571159/258057495-0c399484-5238-4ca6-a103-7179e0683b08.png) As i read that doesnt seem to be needing symbols

The page says the following:

- builds shipped via the Dev channel of the Windows Insider Program
- Which ExplorerPatcher functionality requires the use of symbol files?
Currently, symbol data is required for the following functionality:

twinui.pcshell.dll: Win+X menu
twinui.pcshell.dll: skinning and unskinning some context menus
twinui.pcshell.dll: legacy Windows 10 window switcher
twinui.pcshell.dll: functionality that opens the calendar flyout instead of Microsoft Teams when pressing Win+C (until ExplorerPatcher version 22000.343.41.11)
StartDocked.dll: open Start directly to "All apps"
StartDocked.dll: maximum numbers of apps to show in the "Most used" section
